### PR TITLE
maintenance/reorg-imports

### PR DESCRIPTION
**Pull request by daviduzumeri**

### CODE REVIEW

File: global.d.ts [hunk 1]
```diff
@@ -1,12 +1,5 @@
 import '@emotion/react'
 
-type BaseStyles =
-  | string
-  | number
-  | {
-      [ruleOrSelector: string]: BaseStyles
-    }
-
```
Comment (clayton-duarte) [inline global.d.ts:9]: we dont need it anymore?

Comment (daviduzumeri) [reply global.d.ts:9]: It wasn't even being used!

File: src/gatsby-theme-docz/wrapper.jsx [hunk 2]
```diff
@@ -1,6 +1,6 @@
 import React from 'react'
 
-import ThemeProvider from '../../../src/'
+import ThemeProvider from '../components/ThemeProvider'
```
Comment (clayton-duarte) [inline src/gatsby-theme-docz/wrapper.jsx:3]: I think that it might fail due to some Gatsby dumbness, not sure about it though

**Review by clayton-duarte — APPROVED** [review]
If the build-deploy GHA pass, so the Gatsby change is safe

File: src/gatsby-theme-docz/wrapper.jsx (hunk 2 above)
Comment (daviduzumeri) [reply src/gatsby-theme-docz/wrapper.jsx:3]: Haha it did, one sec

Comment (clayton-duarte) [reply src/gatsby-theme-docz/wrapper.jsx]: yeah, Gatsby is failing on build... updating this path to '../../../src/components/ThemeProvider' might solve the issue (or we can dive deeper on Gatsby and fix the way it resolves paths, but I don't think we need that much)

File: src/gatsby-theme-docz/wrapper.jsx [hunk 3]
```diff
@@ -1,6 +1,6 @@
 import React from 'react'
 
-import ThemeProvider from '../components/ThemeProvider'
+import ThemeProvider from '../../../src/components/ThemeProvider'
```
Comment (clayton-duarte) [inline src/gatsby-theme-docz/wrapper.jsx:3]: heheh, you were faster then me

**Review by clayton-duarte — APPROVED** [review]
🤞

**Review by clayton-duarte — APPROVED** [review]

**Pull request merged**